pkg/config/config.go

```diff
@@ -220,6 +220,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("statsd_forward_host", "")
 	config.BindEnvAndSetDefault("statsd_forward_port", 0)
 	config.BindEnvAndSetDefault("statsd_metric_namespace", "")
```

Comment:
we should have a discussion on deprecating this option (with a warning when used + removing it from documented options in `datadog.yaml`), and potential alternatives users could use